### DISCUSSION TITLE
Feature request: set climate to away/none instead of off/on

### DISCUSSION THREAD

**Discussion by mouth4war** [jseidl/magic-areas#199] [Ideas / Feature Requests]
Can we have a flag for climate entities setting to presets away instead of off, none instead of on? This is useful for heaters.

**Comment by jseidl** [comment]
[Release 3.0.0](https://github.com/jseidl/hass-magic_areas/tree/release/3.0.0-beta) (currently on beta) supports changing climate mode instead of on/off, might solve?

**Comment by mouth4war** [comment]
How do you configure that? Don't believe I saw any option for it.

**Comment by jseidl** [comment]
You don't, it just changes between HVAC_MODE_OFF (which might not be what you want) and HEAT/COOL (whatever is available for the climate device), see https://github.com/jseidl/hass-magic_areas/blob/release/3.0.0-beta/custom_components/magic_areas/climate.py#L450-L473

**Comment by mouth4war** [comment]
I meant preset mode not HVAC mode

**Comment by mouth4war** [comment]
Hi is there any more information required?

**Comment by jseidl** [comment]
No further info. It's on the backlog!

**Comment by jseidl** [comment]
@mouth4war can you check if the new behavior on 3.0 solves this for you?

**Comment by Nuuki9** [comment]
Was this ever added? For sure the ability to change to/from a defined climate preset based on presence would be very useful.

**Comment by jseidl** [comment]
Implemented on https://github.com/jseidl/hass-magic_areas/pull/493
